doc/user/content/guides/cdc-postgres.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Change Data Capture (Postgres)"
+description: "How to create real-time materialized views from PostgreSQL changelog data with Materialize."
+weight: 10
+menu:
+  main:
+    parent: guides
+---
+
+Change Data Capture (CDC) allows you to track and propagate changes in a database to downstream consumers based on its changelog. In this guide, we'll cover how to use Materialize to create and efficiently maintain real-time materialized views on top of Postgres CDC data.
```

Comment:
I've only seen `changelog` being used for what changes were included in a release of a piece of software. If it's not just me I'd switch to `write ahead log`

Reply:
That's a good point! I've seen it used loosely as a "generalization" term (i.e. it's the WAL for Postgres but binlog for MySQL, so "changelog" can be used as a reference to both), but since this is Postgres-specific and I don't really spell out WAL anywhere else, it's a good idea to replace it here!